saltboot/hostname.go

```diff
@@ -108,7 +118,7 @@ func updateHostsFile(hostName string, domain string, file string, ipv4address st
 	hostsFile := string(b)
 	log.Printf("[updateHostsFile] original hosts file: %s", hostsFile)
 
-	ipv4HostString := ipv4address + " " + hostName + domain + " " + hostName
+	ipv4HostString := ipv4address + " " + constructFQDN(hostName, domain) + " " + getShortHostName(hostName, domain)
```

Comment:
Using `fmt.Sprintf` might be little more readable here:
```suggestion
	ipv4HostString := fmt.Sprintf("%s %s %s", ipv4address, constructFQDN(hostName, domain), getShortHostName(hostName, domain))
```